DESCRIPTION.md

```diff
@@ -500,6 +500,30 @@ If you get an error telling you "The request signature is invalid", please check
 - You use the data to sign to create a SHA256 hash signature.
 - You have base64 encoded the SHA256 hash signature before adding it to the request under `X-Bunq-Client-Signature`.
 
+**Note:** Though request signing is a must on production, you can choose to disable it on sandbox to simplify the testing. Here's how it works:
+
+1. Set the `X-Bunq-Client-Signature-Validation-Policy` header of the request to `IGNORE_ONLY_FOR_TESTING`.
+1. Send the request.
+
+When ready to try your integration on production, remove the `IGNORE_ONLY_FOR_TESTING` header setting and implement signing in sandbox first. Make sure it works and then change the base URL to `https://api.bunq.com`.
+
+# <span id="topic-signing">Encryption</span>
+
+Some API calls such as `POST /user/{userID}/card-debit` and `POST /user/{userID}/card-credit` require additional encryption to protect the sensitive data that they pass. 
+
+Here is how to encrypt a request:
+1. Generate a random [Initialization Vector](https://en.wikipedia.org/wiki/Initialization_vector) (IV) of 16 bytes.
+1. Generate a random [Advanced Encryption Standard](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) (AES) key of 32 bytes.
```

Comment:
This is my bad, since I told you wrong, but lets make it `256 bits (32 bytes)` since in terms of encryption we should always talk in bits. (it is practically the same, but makes more sense like this)